### PR TITLE
List structure and Project setup

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
# Hey There, 👍🏻 

> In this project, I:

- Setup the project with Webpack and other dependencies
- Installed and enabled the webpack dev Server
- created the HTML document and stylings for the project
- Dynamically generated the HTML for the input values
- Committed, pushed changes

## Happy Reviewing

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @PromzzyKoncepts 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] I noticed that your README.md file is empty, Please your `README` file should be descriptive - all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present 👌

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- minimalist website
+- etc
```
Comment (zieeco) [inline README.md:64]: - Please, kindly add the license and its content to make your readme more professional.

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+## Built With
+
+- Html,JS,CSS
+- Webpack andother dependencies
+- Technologies used
+
+## Live Demo (Coming Soon)
+
+[Live Demo Link](https://livedemo.com)
+
```
Comment (zieeco) [inline README.md:42]: - Please, also add the "Getting Started" section with instructions on how users can have your project up and running on their machines.

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+## Authors
+
```
Comment (zieeco) [inline README.md]: - Please, kindly make the authors singular as you've worked on this project alone.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+window.onload = onPageLoad();
+
+// document.querySelector('#add').addEventListener('keypress', function (e) {
+//     if (e.key === 'Enter') {
+//       e = taskDisplay()
+//       inputVal.value = ''
+//     }
+// });
```
Comment (zieeco) [inline src/index.js]: - Let's make our codebase clean by removing commented code blocks.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @PromzzyKoncepts,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @PromzzyKoncepts , 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Made requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**